src/app/[locale]/globals.css

```
@@ -22,126 +22,243 @@
+  --surface-rgb: 253, 246, 240;
+  --selector-txt: #22222c;
+  /* Brands */
+  --nextjs-fill: #171717;
+  --html-fill: #000;
+  --github-fill: #0f0e0fff;
+  /* Rocket */
+  --rocket-body: #e3602d;
+  --rocket-secondary: #2c6a9e;
+  --rocket-window: #f3c74d;
+
+  --surface-rgb: 28, 37, 54;
```

Stylelint (17.6.0)

[error] Unexpected duplicate "--surface-rgb" (declaration-block-no-duplicate-custom-properties)

(declaration-block-no-duplicate-custom-properties)